midend/lib/Conversion/MatMulOptimization/CMakeLists.txt

```diff
@@ -1,14 +1,18 @@
 add_mlir_library(MatMulOptimization
-  BatchMatMulOptimize.cpp
 	MatMulOptimize.cpp
   MatMulVectorization.cpp
   MatMulParallelVectorization.cpp
+  BatchMatMulOptimize.cpp
+  BatchMatMulTileOptimize.cpp
+  BatchMatMulSCFOptimize.cpp
```

Comment:
Do our optimized methods need to add into both `MatMulOptimization` and `BatchMatMulOptimization`?